src/modules/test/recipes/compliance_EnsureShadowGroupIsEmpty.jsonc

```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "Action": "LoadModule",
+    "Module": "compliance.so"
```

Comment:
We have one single test recipe per module, and that test recipe has same name as the module. 

Please consider combining these multiple test recipes into a single test recipe.